eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/io/VariantContextWriterTest.java

```diff
@@ -276,4 +297,63 @@ public void throwIfNonStandardNucleotides() throws Exception {
         File output = temporaryFolder.newFile();
         assertWriteVcf(output, buildVariant(CHR_1, 1000, "C", "U", SNP_SEQUENCE_ONTOLOGY, STUDY_1));
     }
+
+    @Test
+    public void checkDefaultFlags() throws Exception {
+        File output = temporaryFolder.newFile();
+        assertWriteVcf(output,
+                       buildVariant(CHR_1, 1000, "C", "G", SNP_SEQUENCE_ONTOLOGY, false, false, true, true, true,
+                                    STUDY_1));
+
+        String dataLinesRegex = "^[^#]";
+        String dataLinesWithFlagsRegex = dataLinesRegex + ".*(" + VALIDATED_KEY
+                                    + "|" + SUBMITTED_VARIANT_VALIDATED_KEY
+                                    + "|" + ALLELES_MATCH_KEY
+                                    + "|" + ASSEMBLY_MATCH_KEY
+                                    + "|" + SUPPORTED_BY_EVIDENCE_KEY + ")";
+        List<String> dataLines = grepFile(output, dataLinesWithFlagsRegex);
+
+        assertEquals(0, dataLines.size());
+    }
```

Comment:
If the purpose of this test is to check whether flags with default values do not appear explicitly in the VCF, please name the test accordingly (checkDefaultFlags conveys a different meaning).